test/unit/action/info.spec.js

```diff
@@ -15,8 +15,7 @@ describe('Action Info Unit Tests', () => {
   beforeEach(() => {
     sandbox = sinon.createSandbox({});
     sandbox.stub(logger);
-    useStrict(false);
-    store = observable({ lndReady: false });
+    store = new Store();
```

Comment:
This refactor is very easy on the eyes :100: makes test writing way easier as well